indexer/README.md

```diff
@@ -1,226 +1,176 @@
-```
-
-### Methods and Endpoints
+    ```
+1. Then, replace the following values in the config.json with your own values.
+    - `<stage_name>` is the intended stage name of the Lambda
+    - `<your-indexer-lambda-application-name>` is the name that you will give the Lambda project.
+    - `<AWS-account-ID>` is one of the values in the access policy of your Elasticsearch. An numerical id of your AWS Account. 
+    - `<your blue box>` is the endpoint of the HCA DSS.
+    - `<your elasticsearch endpoint>` is url you will use to access Azul's Elasticsearch Domain. This url can be found by returning to your Elasticsearch Service Dashboard,
+    clicking the link of your Elasticsearch Domain. The url should be labled as **Endpoint** in the Overview Tab.
+
+### 4. Modifying IAM Access Policies
+
+1. In the AWS Console, click Services in the top toolbar, and click IAM in the Security, Identity & Compliance subsection.
+1. On the side menu bar, click **Roles**, then search for your Lambda function, `<your-indexer-lambda-application-name>`. Once you find it, click on it.
+1. Find a table in the Permissions tab under the blue Attach Policy button. You should see for Lambda name in the table. Click on the caret on the right side of the Lambda name. Then, click on "Edit Policy".
+1. Add the policy found in `policies/elasticsearch-policy.json` file in the cgp-dss-azul-indexer repository (Make sure to change the `Resource` value to the ARN of your elasticsearch box).
+
+### 5. Deploying Indexer Lambda
+
+1. In the directory of your chalice function, run
+   ```
+   chalice deploy --no-autogen-policy
+   ```
+   **Note:** If you are deploying an AWS API Gateway stage other than the default (dev)
+add the `--stage <stage name>` option. Since we have created a policy in AWS we do not want chalice to automatically create a policy for us.
+
+### 6. Registering a Subscription for Notification with the DSS
+
+1. Make sure your computer has a working web browser.
+1. If you are currently in a virtual environment, run the command `deactivate` to get out of the environment.
+1. Create a new Python 3.6 virtual environment and activate it.
+1. Install HCA using the command
+   ```
+   pip install --upgrade hca
+   ```
+1. Setting up config file using the instructions located [here](https://github.com/HumanCellAtlas/dcp-cli#development).
+1. Check if your HCA DSS deployment has been configured correctly by running
+   ```
+   hca dss post-search --replica aws --es-query '{}'
+   ```
+You should receive a response json containing one or more urls. Check if the returned URLs match the intended DSS Server.
+1. Login to HCA using the command
```

Comment:
Let's put the "dss login" step before the "dss post-search" step, so that when we deploy the DSS with all the APIs requiring authentication, this will still work.